src/chapter1/01-hello-world.md

```diff
@@ -98,7 +103,26 @@ Alternatively you can type "cmd" into the search field. An app called "command p
 
 ![](images/search-field.png)
 
-No matter how you do it, you should now stare at a _beautiful_ command line. Take a second and _bask in it's glory_ for a while.
+#### Installing Node.js on macOS
+
+Grab Node.js from `https://nodejs.org`:
+
+![](images/node-js-macos.jpeg)
+
+> Note that by the time you are reading this book the Node.js version you see on `https://nodejs.org` might be different.
+
+You want to download the LTS (long term support) version. Click the big green button with the _LTS_ version. This will download a pkg file. Execute the pkg file by double-clicking it. You will be presented with the following screen:
+
+![](images/node-js-macos-installer.jpeg)
+
+You will be presented with a bunch of options during the installation - just click "Continue" everywhere and you will be fine.
+
+Now we need to open a **command-line interface** (also called command line, command prompt or CLI) on our computer. A command line allows you to execute various tasks (called commands) on your computer. On a Mac you can open the command line by hitting <kbd>Cmd</kbd> + <kbd>Space</kbd> to open _search_, typing _terminal_ and hitting <kbd>Return</kbd>.
```

Comment:
This is potentially confusing for people having a rough understanding of the difference between "Terminal", "Shell", "Prompt" and "Command Line Interface".